src/LanguageServer/Protocol/Handler/Diagnostics/AbstractWorkspacePullDiagnosticsHandler.cs

```diff
@@ -79,6 +81,11 @@ private void OnLspTextChanged(object? sender, EventArgs e)
         UpdateLspChanged();
     }
 
+    private void OnWorkspaceRefreshRequested()
+    {
+        UpdateLspChanged();
+    }
```

Comment:
@dibarbet  to review.  we probably want docs here.  in particular, it's unclear why we need this given other systems (like EnC) which request a refresh, but not need to do this.

Reply:
I can't remember if EnC does workspace diagnostics, if they only do doc diagnostics this does not affect them.  If they do workspace diagnostics, they definitely need this.

Reply:
➡️ Removed from this pull request